templates/alerts_logger.py.j2

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+"""Tendrl Notifier REST API logger."""
+
+import logging
+import os
+import json
+import time
+
+import requests
+
+
+URL = "http://0.0.0.0/api/1.0/"
+USER = "{{ tendrl_user }}"
+PASSWORD = "{{ tendrl_password }}"
```

Comment:
Same as above, wouldn't be better, to read those parameters from command line arguments or some configuration file?